mailbox/store/src/main/java/org/apache/james/mailbox/store/quota/ListeningCurrentQuotaUpdater.java

```diff
@@ -111,9 +111,12 @@ private void handleAddedEvent(Added added, QuotaRoot quotaRoot) throws MailboxEx
     }
 
     private void handleMailboxDeletionEvent(MailboxDeletion mailboxDeletionEvent) throws MailboxException {
-        currentQuotaManager.decrease(mailboxDeletionEvent.getQuotaRoot(),
+        boolean mailboxContainedMessages = mailboxDeletionEvent.getDeletedMessageCount().asLong() > 0;
+        if (mailboxContainedMessages) {
+            currentQuotaManager.decrease(mailboxDeletionEvent.getQuotaRoot(),
```

Comment:
It looks like indentation issues

Reply:
4 space indented instead of 8 : that's ok :-)

BTW it passed the checkstyle ;-)

Reply:
:)